README.md

```diff
@@ -10,9 +10,10 @@
 
 # Features
 
-- Create users
-- Creat posts
-- Comment posts
+- Login.
+- Create a new user.
+- Create posts.
+- Interact with posts, commenting on them and liking them.
 
```

Comment:
### Optional
- [ ] Since this is an API project, it would be better to list out the routes to be tested in our README file for simplification. For example;
```
Login endpoint - POST : /users/sign_in

Registration endpoint - POST : /users/signup
```